src/containers/meeting/third-party/CreateModal/index.tsx

```
@@ -1,10 +1,45 @@
+  onDone
+}: CreateModalProps) {
+  const client = useApollo();
+  const { success, error } = useAlertStore();
+
+  // 공통 상태
+  const [loading, setLoading] = useState(false);
+
+  // 모임 생성 관련 상태
+  const [groupName, setGroupName] = useState("");
+  const [groupDescription, setGroupDescription] = useState("");
+  const [groupIntroduce, setGroupIntroduce] = useState("");
```

ESLint

[error] 'groupIntroduce' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)